javascript/ql/src/semmle/javascript/security/dataflow/TaintedPathCustomizations.qll

```diff
@@ -614,4 +619,221 @@ module TaintedPath {
   class SendPathSink extends Sink, DataFlow::ValueNode {
     SendPathSink() { this = DataFlow::moduleImport("send").getACall().getArgument(1) }
   }
+
+  /**
+   * Holds if there is a flow step from `src` with flow-label `srclabel` to
+   * `dst` with flow-label `dstlabel` for a tainted-path vulnerability.
+   */
+  predicate isAdditionalTaintedPathFlowStep(
+    DataFlow::Node src, DataFlow::Node dst, DataFlow::FlowLabel srclabel,
+    DataFlow::FlowLabel dstlabel
+  ) {
+    isTaintedPathStep(src, dst, srclabel, dstlabel)
```

Comment:
Would it make sense to merge these two predicates now? Otherwise, I'd suggest renaming the other one to `isPosixPathStep` or something like that. Right now the naming and separation is a little weird.